docker/types/services.py

```diff
@@ -659,10 +659,12 @@ class Placement(dict):
                 are provided in order from highest to lowest precedence and
                 are expressed as ``(strategy, descriptor)`` tuples. See
                 :py:class:`PlacementPreference` for details.
+            maxreplicas (int): Maximum number of replicas per node
             platforms (:py:class:`list` of tuple): A list of platforms
                 expressed as ``(arch, os)`` tuples
     """
-    def __init__(self, constraints=None, preferences=None, platforms=None):
+    def __init__(self, constraints=None, preferences=None, maxreplicas=None,
```

Comment:
Can you add the new argument at the end please? This avoids breaking calls with positional args.

Reply:
Fixed as per your request, not sure why one of the tests failed this time